beeswax/augment/augmentor.proto

```diff
@@ -23,20 +24,34 @@ message AugmentorRequest {
 //
 // Also custom bidding agents can receive this data through the augmentor_data field in
 // the BidRequestExtension.
+// Next id: 3
 message AugmentorResponse {
 
+  // Next id: 4
   message Segment {
     // The Id of an augmentor segment.
     // This field is primarily used for line item targeting.
     // For targeting to work with the segment ids, it is necessary
     // to register the segment ids with Beeswax via the Buzz Segments API.
     optional string id = 1;
 
-    // This field can be used to pass additional information about the segment
-    // that Stinger will forward to the bidding agent.
-    optional string value = 2;
+    // This is a deprecated field which is not used any more.
+    optional string deprecated_string_value = 2 [deprecated = true];
+
+    // This field is ONLY used for bid modifiers as a dynamic mulitplier. If
+    // the segment is present in the request, the multiplier will be applied to
+    // the bid price. A valid range of multipliers should be between 0 and 10
+    // and invalid ones will be ignored.
+    optional float value = 3;
   }
 
   // Array of Augmentor segments
   repeated Segment segments = 1;
+
+  // Next id: 3
+  message Macro {
+    optional string name = 1;
+    optional string value = 2;
+  }
+  repeated Macro dynamic_macros = 3;
```

Comment:
shall we make this tag to be 2 since we made the comment that next id is 3?

Reply:
ya good catch. i think that was a typo in the other branch as well